pkg/api/logging/v1alpha2/types.go

```diff
@@ -27,8 +27,6 @@ type APIResponse struct {
 
 type Query struct {
 	Operation       string
-	WorkspaceFilter string
```

Comment:
workspacesearch filter and workspacesearch are both deleted?
log searching UI doesn't have workspace search condition?

Reply:
search log with workspace is a relatively common demand，it should be added in 3.1，this can be kept first

Reply:
Workspace filter has never been used since 2,1.1. Namespace filter is enough to cover all use cases.

I think we can remove workspace filter until we see increasing demand. We have too many query parameters in logging APIs, this may confuse developers.

![image](https://user-images.githubusercontent.com/22350668/88869147-9a815e00-d244-11ea-941f-a9f19d946f7d.png)

Reply:
ok, agree